### DISCUSSION TITLE
Error mounting known_hosts file when starting Docker Compose application

### DISCUSSION THREAD

**Discussion by yohangwak** [markshust/docker-magento#1107] [Q&A] [answered]
Hi,
I get this error when attempting to start docker magento

Cannot start Docker Compose application. Reason: compose [start] exit status 1. Container magento-rabbitmq-1 Starting Container magento-redis-1 Starting Container magento-db-1 Starting Container magento-opensearch-1 Starting Container magento-mailcatcher-1 Starting Container magento-mailcatcher-1 Started Container magento-opensearch-1 Started Container magento-rabbitmq-1 Started Container magento-db-1 Started Container magento-phpmyadmin-1 Starting Container magento-redis-1 Started Container magento-redis-1 Waiting Container magento-opensearch-1 Waiting Container magento-rabbitmq-1 Waiting Container magento-db-1 Waiting Container magento-phpmyadmin-1 Started Container magento-opensearch-1 Healthy Container magento-redis-1 Healthy Container magento-db-1 Healthy Container magento-rabbitmq-1 Healthy Container magento-phpfpm-1 Starting Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/host_mnt/Users/yohan/.ssh/known_hosts" to rootfs at "/var/www/.ssh/known_hosts": mount /host_mnt/Users/yohan/.ssh/known_hosts:/var/www/.ssh/known_hosts (via /proc/self/fd/6), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

I think I have correct permission

ls -la /Users/yohan/.ssh
total 40
drwx------@  7 yohan  staff   224 Apr  8 19:35 .
drwxr-x---+ 26 yohan  staff   832 Apr  9 18:54 ..
-rw-------   1 yohan  staff   145 Apr  8 17:41 config
-rw-------@  1 yohan  staff  3389 Mar 17 11:20 id_rsa
-rw-r--r--@  1 yohan  staff   746 Mar 17 11:20 id_rsa.pub
-rw-------@  1 yohan  staff  1446 Apr  6 10:30 known_hosts
-rw-------@  1 yohan  staff   501 Apr  5 17:56 known_hosts.old

I tried to comment out those but still get same error

 volumes: &appvolumes
      - ~/.composer:/var/www/.composer:cached
      #- ~/.ssh/id_rsa:/var/www/.ssh/id_rsa:cached
      #- ~/.ssh/known_hosts:/var/www/.ssh/known_hosts:cached

**Comment by its-leoeff511** [comment, marked as answer]
I ran `bin/remove` and then `bin/start` and it worked for me

**Reply by its-leofisher** [reply]
also noticed this occurs more often if i don't run `bin/stopall` before `bin/start` and even at times `bin/restart`

Rebooting Docker desktop resolves it as well.

**Reply by its-leofisher** [reply]
just for completeness, I have been running `bin/start --no-dev` and then `bin/start`, usually works.  `bin/remove` should be a last resort.
